analytics/src/analytics/integrations/etldb/migrations/versions/0005_add_proj_col_to_issue_hist.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE IF EXISTS gh_issue_history ADD COLUMN IF NOT EXISTS project_id INTEGER DEFAULT 0;
```

Comment:
Also two notes on the `project_id` column specifically:
- `project_id` should probably be non-nullable
- Even if we make it non-nullable, I'm not sure we want to set a default in perpetuity.

I'd rather it raise an error so we don't silently insert values that aren't correctly mapped to a project moving forward. I'd rather it fail at runtime and halt the batch process, than discover there's a bug several days later only because the downstream metrics start to drift (I've experienced this pain far too many times).

We either want to remove the default value after creating the column, or create it as optional, insert a dummy value, then alter it to make it required.

Reply:
Agreed on `project_id` being a not null column. I'll push a commit in a moment.

Re: table name, I recommend keeping it simple. The table is in fact a history of issues and their relationships etc.  :)

Reply:
Thanks for fixing that in [this commit](https://github.com/HHS/simpler-grants-gov/pull/2992/commits/68f20a389abed5c27b8c8074d169e7682cbec5f9)